src/main/webapp/index.html

```diff
@@ -22,7 +22,14 @@ <h1>
           Discover the world's hidden gems <br />
           before planning your next trip.
         </h1>
-        <input type="text" name="search" placeholder="Search a city, country, etc." />
+        
+        <!-- Search Bar -->
+        <form autocomplete="off">
+          <div class="autocomplete">
+            <input id="myInput" type="text" name="Country" placeholder="Search a country" />
+          </div>
+          <input type="submit" value="Go" />
```

Comment:
Maybe try this?
```suggestion
          <button type="button" onclick="search()">Go"</button>
```

Then in your javascript file you'll need something like:
```javascript
function search() {
  const query = document.getElementById('myInput').innerText;
  window.location.href = "/SearchResults.html?q=" + encodeURIComponent(query);
}
```

You'll also want to update the behavior when `Enter` is pressed so this function is called.